_includes/live-demos/file-picker/index.js

```diff
@@ -1,5 +1,5 @@
 tinymce.init({
-  selector: '#editor',
+  selector: '#file-picker',
```

Comment:
For consistency?
```suggestion
  selector: 'textarea#file-picker',
```

Reply:
I also saw this, but didn't want to make this change as I assumed it wasn't done for a specific reason 🤷

Reply:
> I assumed it wasn't done for a specific reason

probably not. it's pretty inconsistent. both are valid, but I prefer the explicit version.